test/scripts/download-latest-rt-binaries.sh

```diff
@@ -167,11 +212,25 @@ mkdir -p $OUTPUT_PATH
 # Download requested binaries
 if $DOWNLOAD_TANSSI_NODE; then
   echo "Fetching tanssi-node from $TANSSI_IMAGE..."
+  BINARIES=(
+    tanssi-node
+  )
+  BINARIES_SKYLAKE=(
+    tanssi-node-skylake
+  )
+  BINARIES_ZNVER3=(
+    tanssi-node-znver3
+  )
+  if [ "$INCLUDE_ALL_ARCHS" = true ]; then
+      BINARIES+=("${BINARIES_SKYLAKE[@]}")
+      BINARIES+=("${BINARIES_ZNVER3[@]}")
+  fi
   docker run --rm \
-      --entrypoint tar \
-      "$TANSSI_IMAGE" \
-      -C /tanssi -cf - tanssi-node \
-    | tar -C $OUTPUT_PATH -xf -
+    --entrypoint tar \
+    "$TANSSI_IMAGE" \
+    -C /tanssi -cf - \
+      "${BINARIES[@]}" | tar -C $OUTPUT_PATH -xf -
+
   chmod +x $OUTPUT_PATH/tanssi-node
   echo "→ $OUTPUT_PATH/tanssi-node"
```

Comment:
This could chmod and echo the real $BINARIES, although chmod is only needed for tests so it shouldn't matter

Reply:
that is why I left it like this...not important